recipes/default.rb

```diff
@@ -275,17 +275,11 @@
     notifies :reload, 'service[tinc]', :delayed
   end
 
-  # We need this for systemd configuration starting from debian-stretch
+  # We need this for systemd configuration
   # /etc/tinc/nets.boot are no longer working / is ignored, see https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=841052#27
-  if node['platform'] == 'debian'
```

Comment:
we really need to understand why we had the debian guard - is it just a leftover, planning ahead or did we run this on other systems and those had no unit support?

Reply:
You added it from this commit https://github.com/EugenMayer/chef-tinc-cookbook/commit/0d4471cb8f55e6cb2afd41dede763c2e903d8c46#diff-55bf87238c9b8af164c5133a60721f12.

Then I've updated the way to detect the Debian version because of Debian Jessie (we've talked about it in #11). Since I've migrated to Buster.